test_app/fixtures/callable.py

```diff
@@ -0,0 +1,2 @@
+def my_callable(context):
+    context.test_me = 'hi!'
```

Comment:
What does this callable do useful for the test?

Shouldn't it create an object (in the database)?